styles.css

```diff
@@ -0,0 +1,26 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+  background-color: mediumturquoise;
+}
+
+section {
+  background-color: aliceblue;
+  height: 130px;
+  width: 500px;
+  border-radius: 5px;
+}
+
+h4 {
+  text-align: center;
+}
+
+h3 {
+  display: flex;
+  justify-content: center;
```

Comment:
You don't need a flexbox for that, since you have no child elements, but only text content. So you could use `text-align: center` here for the same styling result, like you did it already above, or?